### DISCUSSION TITLE
Android client: please add a tts event to announce if someone has intercepted you, this is causing serious problems

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#2015] [Ideas]
### Description

once i was having a private conversation with somebody in a server with my android device because i was out of the house and i didn't have any computer or laptop out with me, and later i found out that i got intercepted by a random administrator on that server and he recorded my conversation using Virtual Recorder, with out me knowing anything about it
so many times i've heard so many complanes about some teamtalk servers because apparently there are some cases where people has been reported that they were with their android client and having an online sexual with their partner and they got intercepted with out being notifyed about it, and later that has been turned to a huge problem for them
and there are a lot of other cases like that, i just mentioned about this very sensitiv case so you can take care of this, because if someone is using teamtalk, they should be sure that they will be safe for them: intercepting is a really nice feature and i personally use it for a lot of good and useful ways, but like windows client, it shouldn't happen with out user being notifyed: if i know you are intercepting me, that would  be something useful: thanks.
@beqabeqa473 @bear101 @CoBC i really do hope that you guys take care of this: thanks, and i wish the best for you.

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by muamel1** [comment]
lol This is what happens when you go to an untrusted server,
that's true, this feature should be added to mobile clients, not just Android, but also iPhone lacks this feature.

**Comment by amirmahdifard** [comment]
and people are also complaning that when we open issues in tt github, even serious problems, they don't get any results. @bear101 i'm saying this just to help you, a lot of people have stopped using tt or they are stopping, because they say we don't have computer, and mobiles tt isn't safe for us to be able to use it: and they said we will only reuse tt again if these serious problems have been resolved
